apps/chat/src/components/Chat/MessageAttachment.tsx

```diff
@@ -326,9 +364,9 @@ export const MessageAttachment = ({ attachment, isInner }: Props) => {
             (attachment.type === chartType ? (
```

Comment:
@denys-kolomiitsev , custom renderer should has higher priority than even Plotly